.travis.yml

```diff
@@ -28,7 +29,6 @@ addons:
     - libboost1.55-all-dev
     - libsndfile1-dev
     - libqt4-dev
-    - libgtest-dev
```

Comment:
it's better to build for a release, not from master branch of repository ... why changing that ?